config.json

```diff
@@ -537,7 +537,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+       {
+      "title": "Cloud native",
+      "content": "Network primitives in the language make it simpler to write services and run them in the cloud.",
+      "icon": "cloud-native"
+       },
+       {
+      "title": "Flexibly typed",
+      "content": "Structural types with support for openness are used both for static typing within a program and for describing service interfaces.",
+      "icon": "flexible-typing"
+       },
+       {
+      "title": "Data oriented",
+      "content": "Type-safe, declarative processing of JSON, XML, and tabular data with language-integrated queries.",
+      "icon": "data-transformation"
```

Comment:
Name this as data-oriented and change the other PR too accordingly.

Reply:
The hyphen should be added when the term is added as an adjective (e.g., `data-oriented language`).